# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1799495-1.html
# Commit: c5112a80ad2f
# Full Hash: c5112a80ad2f910f06889c9cadecc8c4728cd858
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-11-08 21:36:02
# Description:
#   Bug 1799495 - Ensure WaitForShmem has a valid DrawTargetWebgl to access. r=aosmond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161554
# ==============================================================================

diff -r 879b0989791c -r c5112a80ad2f gfx/tests/crashtests/1799495-1.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1799495-1.html	Tue Nov 08 16:23:56 2022 +0000
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script>
+      window.addEventListener('load', async () => {
+        const canvas1 = document.createElement('canvas')
+        document.documentElement.appendChild(canvas1)
+        const context1 = canvas1.getContext('2d')
+        context1.globalCompositeOperation = 'destination-atop'
+        const path = new Path2D()
+        context1.fill(path, 'nonzero')
+        const canvas2 = document.createElement('canvas')
+        const context2 = canvas2.getContext('2d', {})
+        await window.createImageBitmap(canvas1, 124, 233, 2147483647, 1, {})
+        context2.filter = 'invert( 90% )'
+        context2.fillRect(-1, -128, 4096, 1024)
+      })
+    </script>
+</head>
+</html>
\ No newline at end of file